### DISCUSSION TITLE
CrazyKhoreia

### DISCUSSION THREAD

**Discussion by santiagorg2401** [USC-ACTLab/crazyswarm#657] [Show and tell]
Hi everyone,
I would like to  show you my thesis project called "CrazyKhoreia", it's an easy to use tool to generate trajectories from digital images that may be used for light painting or multiple UAVs formations, its link is https://github.com/santiagorg2401/crazyKhoreia, I hope you like it.

**Comment by whoenig** [comment]
Awesome - thanks for sharing! Are the generated trajectories then executed using the Crazyswarm?

**Reply by santiagorg2401** [reply]
They should work regardless of the UAV (I even tried those on a terrestrial one for drawing with a pen though), of course the real dynamics and flight quality may affect the final results, however, I tested everything with the Crazyswarm, I also built a GUI, https://github.com/santiagorg2401/crazygui, I need some final adjustments on that one and of course better documentation for easier installation and a pull request in the future because I had to add a simple launch file in Crazyswarm.
